src/test/java/formatter/FormatterTest.java

```diff
@@ -0,0 +1,36 @@
+package formatter;
+
+import cz.codecamp.logger.Formatter;
+import cz.codecamp.logger.LogLevelEnum;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FormatterTest {
+
+    private Formatter formatter;
+
+    @Before
+    public void setup(){
+        formatter = new Formatter();
+    }
+
+    @Test
+    public void testLogLevel(){
+        String message = formatter.format(LogLevelEnum.DEBUG, "TestMessage");
+        Assert.assertTrue(message.contains("[" + LogLevelEnum.DEBUG.name() + "]"));
```

Comment:
Doporučuji používat `assertThat()` (viz slidy od Jirky Pénzeše) - pokud test failne, dá mnohem lepší message, proč vlastně test spadnul. Tohle akorát řekně něco jako "failed asserting that false is true".